packages/editor/src/components/EditorContainer.tsx

```diff
@@ -246,17 +247,26 @@ const EditorContainer = () => {
         })}
       </div>
       <PopupMenu />
-      {!isWidgetVisible && initialized && (
```

Comment:
any reason why we removed these conditions?

Reply:
removing `isWidgetVisible` is causing an issue where both the widget and the button are visible simultaneously. When the widget is displayed, the button should be hidden to avoid overlap

![image](https://github.com/user-attachments/assets/60389ee9-7848-4363-9889-6365f19840ab)


`initialized` flag ensures that the widget is fully initialized and the chat can be opened. If removed, the button will become unresponsive when the Zendesk script hasn’t fully loaded or if the Zendesk configuration is missing, incomplete, or disabled.

Reply:
i removed it for testing, im sure i reverted that, ill add it back if its not in there

Reply:
updated